doc/source/rllib.rst

```diff
@@ -68,6 +68,35 @@ The most important options are for choosing the environment
 with ``--env`` (any OpenAI gym environment including ones registered by the user
 can be used) and for choosing the algorithm with ``--run``
 (available options are ``PPO``, ``A3C``, ``ES`` and ``DQN``).
+In order to save checkpoints from which to restore training 
```

Comment:
Make this a subsection ("Evaluating trained agents") and move to below the specifying params section below?